src/services/common.ts

```diff
@@ -0,0 +1,66 @@
+import { Between, FindManyOptions, Repository } from 'typeorm';
+
+// import { NettotalsEntity } from '../entity/nettotals.entity';
+import { IGetLimitParams } from '../types/query-request';
+import { getStartPoint } from '../utils/period';
+
+// type TRepository = NettotalsEntity;
+
+export async function getLimitQuery<T>({
```

Comment:
`common.ts` and `getLimitQuery` are not descriptive names. Consider renaming to `chartData.service.ts` and `getChartData`.

Reply:
I moved it.